src/frontend/src/lib/types/network.ts

```diff
@@ -1,10 +1,16 @@
+import type { OnRamperNetworkId } from '$lib/types/onramper';
+
 export type NetworkId = symbol;
 
 export type NetworkEnvironment = 'mainnet' | 'testnet';
 
-export interface Network {
+export type Network = {
 	id: NetworkId;
 	env: NetworkEnvironment;
 	name: string;
 	icon?: string;
+} & NetworkLinkedData;
+
+export interface NetworkLinkedData {
+	onRamperId?: OnRamperNetworkId;
```

Comment:
Should we forseen that we may support more links to crypto exchange than OnRamper?

Reply:
you mean to create a sub interface inside NetworkLinkedData?

```
interface OnRamper {
   onRamperId...
}
```

Reply:
I just meant what if we soon add another crypto exchange XYZ? then we just add xyzId next to onRamperId?

Reply:
We already use exchange to get the conversion Token / USD. Should we convert this also as exchange or those are two different things?

Reply:
well, this one is specific for networks. For token I would have merged with `exchange`

Reply:
So not using exchange here? Like `NetworkOnrampData` or `NetworkFiatToCryptoData` or something?

I think I gonna be confuse if there are exchange and exchange, if this makes sense?

Reply:
Sorry, I think I got lost: you would then create a new interface specifically for OnRamper like `NetworkOnramperData`? And put it directly to the `Network` type, or create another interface that collects all the "exchange" data?

Reply:
Something that does not contains the word "exchange", for example:

```
export type Network = {
   ...existingFields
   buy?: NetworkBuyCrypto
}

export type NetworkBuyCrypto {
   onRamperId: OnRamperNetworkId;
}
```

Reply:
Check my last commit please (even if I am not sure if I should create a middle interface or not)

Reply:
Looks good. Hope it works for you too?

Reply:
yes yes! i was afraid on naming, but then all good!